ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -61,7 +61,7 @@
     private static final By PROVIDER = By.id("w1");
     private static final By LOCATION = By.id("w3");
     private static final By WHO_WHEN_WHERE = By.id("who-when-where");
-    private static final By REQUEST_APPOINTMENT = By.linkText("Request Appointment");
+    private static final By REQUEST_APPOINTMENT = By.cssSelector("a[id='appointmentschedulingui.requestAppointmentDashboardLink']");
```

Comment:
Refactored the element attribute `value property`  to a better element reference!